### PR TITLE
Add poetry to travis

### PR DESCRIPTION
**Pull request by gabechu**
# Description
Add poetry install to travis.

# Checklist
- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

### CODE REVIEW

**Comment by gabechu** [comment]
This PR is created for testing purpose, it tests whether Travis has been integrated to the repo. Also testing github configuration, which requires one reviewer at least. See if these two things work properly.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+language: python
+dist: xenial
```
Comment (kerrychu-zen) [inline .travis.yml:2]: we can use the latest distribution `bionic`(refer doc [here](https://docs.travis-ci.com/user/reference/overview/))

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+branches:
+  only:
+    - "master"
+
+before_install:
+  - pip install poetry
+
+install:
+  - poetry install
```
Comment (kerrychu-zen) [inline .travis.yml:19]: great to see poetry here!

File: .travis.yml (hunk 1 above)
Comment (gabechu) [reply .travis.yml:2]: Cool. Will have it updated in this #3 PR.

**Pull request merged**